### PR TITLE
Update Ayu Adiati's member info

### PR DESCRIPTION
**Pull request by adiati98**
## Linked Issue

#13 

## Description

Update Ayu Adiati's info on the member page:
- Update name
- Add mainURL
- Add accounts

### CODE REVIEW

File: src/_data/members/members/adiati98.js [hunk 1]
```diff
@@ -7,24 +7,31 @@ module.exports = {
   //
   // name - if not defined here, will default to your Name on github if defined, if not, then your username
   // name: 'Your Name',
+  name: 'Ayu Adiati'
   //
   // mainUrl: the url your name links to on the members page. defaults to your github profile
   // mainUrl: 'https://virtualcoffee.io',
   //
+  mainUrl: 'https://adiati.com'
```
Comment (danieltott) [inline src/_data/members/members/adiati98.js:15]: I think we need a comma here!

**Review by danieltott — CHANGES_REQUESTED** [review]
Thanks @adiati98 !! One little syntax change and we'll get you merged!

File: src/_data/members/members/adiati98.js (hunk 1 above)
Comment (adiati98) [reply src/_data/members/members/adiati98.js:15]: Oops!!!!
Going to fix it! 

I was in panic mode for a moment there when all tests failed, @danieltott  🙈

**Comment by adiati98** [comment]
Done! Thank you for the help, @danieltott!

**Comment by danieltott** [comment]
image

**Comment by adiati98** [comment]
thank you gif

**Pull request merged**